### PR TITLE
refactor: new app `character` v3.5.0, change app code into `change-character`

### PR DESCRIPTION
**Pull request by 9-9-9-9**

### CODE REVIEW

File: src/main/java/bh/bot/Main.java [hunk 1]
```diff
@@ -79,7 +74,8 @@ public static void main(String[] args) {
 					TrialsApp.class, //
 					GvgApp.class, //
 					GauntletApp.class, //
-					GenMiniClient.class,
+					GenMiniClient.class, //
+					ChangeCharacterApp.class,
```
Comment (9-9-9-9) [inline src/main/java/bh/bot/Main.java:78]: @daredoes you forgot this to make app works

Comment (9-9-9-9) [reply src/main/java/bh/bot/Main.java:78]: Btw how do you think if we change the app code from `character` to `change-character` to make it more meaning full?

**Comment by 9-9-9-9** [comment]
My IDE also detected
```java
while (!characterLoaded && !masterSwitch.get()) {
```
The condition `!characterLoaded` is always true

```java
if (clickImage(BwMatrixMeta.Metas.Character.Dialogs.loading)) {
                            debug("Character Loading!");
                            characterLoaded = true;
                            break;
                        }
```
`characterLoaded = true` the value ` true` assigned to `characterLoaded` is never used

File: src/main/java/bh/bot/Main.java (hunk 1 above)
Comment (daredoes) [reply src/main/java/bh/bot/Main.java:78]: Oops! I refactored this app out from a branch where I had both apps being developed, and forgot this line!

Totally open to changing the character code. I've tried not to spend too much time getting lost in the nitty gritty of making the feature work, as the goal is to play the game not have it play me 😄

**Review by daredoes — APPROVED** [review]
These changes look good to me. We could change the code before merging this though.

**Comment by 9-9-9-9** [comment]
> My IDE also detected
> 
> ```java
> while (!characterLoaded && !masterSwitch.get()) {
> ```
> 
> The condition `!characterLoaded` is always true
> 
> ```java
> if (clickImage(BwMatrixMeta.Metas.Character.Dialogs.loading)) {
>                             debug("Character Loading!");
>                             characterLoaded = true;
>                             break;
>                         }
> ```
> 
> `characterLoaded = true` the value ` true` assigned to `characterLoaded` is never used

@daredoes please double check this as well. I don't have a machine that can test rn. But I believe this new app already works as you expected, right?

File: src/main/java/bh/bot/Main.java (hunk 1 above)
Comment (9-9-9-9) [reply src/main/java/bh/bot/Main.java:78]: Changed app code into `change-character`

**Comment by daredoes** [comment]
> > My IDE also detected
> > ```java
> > while (!characterLoaded && !masterSwitch.get()) {
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The condition `!characterLoaded` is always true
> > ```java
> > if (clickImage(BwMatrixMeta.Metas.Character.Dialogs.loading)) {
> >                             debug("Character Loading!");
> >                             characterLoaded = true;
> >                             break;
> >                         }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > `characterLoaded = true` the value ` true` assigned to `characterLoaded` is never used
> 
> @daredoes please double check this as well. I don't have a machine that can test rn. But I believe this new app already works as you expected, right?

This works as expected. I've been running it all day. I'll try to work on making some VM script to run this, as I think that's the best way to work around the keyboard/mouse capture.

**Comment by 9-9-9-9** [comment]
> This works as expected. I've been running it all day. I'll try to work on making some VM script to run this, as I think that's the best way to work around the keyboard/mouse capture.

👍 I had been using VM to run multiple accounts as well

**Pull request merged**